presentation/invalid/ttml2-prstn-font-embedding-family-invalid.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tt xml:lang="" xmlns="http://www.w3.org/ns/ttml"
+    xmlns:ttm="http://www.w3.org/ns/ttml#metadata"
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter" ttp:profile="ttml2-presentation" ttp:validationAction="warn"
+    xmlns:tts="http://www.w3.org/ns/ttml#styling">
+  <head>
+    <metadata>
+      <ttm:desc>invalid embedded font family name - matches generic font family</ttm:desc>
+    </metadata>
+    <resources>
+      <font src="../resources/test.otf" family="serif"/>
```

Comment:
The referenced font does not exist, so this test could generate the right result for the wrong reason: a processor might mark the file as invalid not because the `family` name is invalid, but because the referenced font does not exist. Happily, we do have a serif font available in the repo already. Suggested change:

```suggestion
      <font src="../resources/fonts/ScheherazadeRegOT.ttf" family="serif"/>
resources/fonts/NotoSans-Regular.ttf
```

Reply:
ok, i'll do something of this sort